cmake/modules/AddCephTest.cmake

```diff
@@ -23,6 +23,21 @@ function(add_ceph_test test_name test_path)
     PROPERTY TIMEOUT ${CEPH_TEST_TIMEOUT})
 endfunction()
 
+function(add_test_label test_name label)
+    get_property(loc_label TEST ${test_name} PROPERTY LABELS)
+    if("${loc_label}" STREQUAL "" )
+        set_tests_properties(${test_name} PROPERTIES LABELS "${label}")
+    else()
+        set_tests_properties(${test_name} PROPERTIES LABELS "${loc_label};${label}")
+    endif()
+endfunction()
+
+# put labels on tests which can be referenced by ctest -L <regex>
+function(add_ceph_test_label test_name test_path test_label)
+  add_ceph_test(${test_name}  "${test_path}")
```

Comment:
@tchaikov, 
I decided to avoid adding optional parameters, but if that is your prefered way, I'll look into it.
Disadvantage of this is that there is only one property type to easily allow to use this model.
When (also) going for COST, parsing extra parameters gets a lot more complicated.

Reply:
> there is only one property type to easily allow to use this model.

i don't understand how you went to this conclusion.

> When (also) going for COST, parsing extra parameters gets a lot more complicated.

i think `cmake_parse_arguments()` renders this task very simple. and the example i put in my comment shows how it works.

Reply:
@tchaikov
It turned out to be easier than I expected.
What I could not find in the docs is that `cmake_parse_arguments()` and fixed position args work together very well. But since they do, this PR is rather straight forward.